README.md

```diff
@@ -38,7 +38,7 @@ Other commands are similarly overridden, e.g. `comment-dwim`: see
 
```

Comment:
Should line 31 also be reworded to use "cut" instead of "copied"?
"type `C-w` then the current line will be ~copied~cut".